### ISSUE TITLE
Can't load PXRDIP .gel images

### ISSUE THREAD

**Issue by wehrenberg1** [HEXRD/hexrdgui#1387]
I get an error,  "can't load zero length imageseries" when trying to load PXRDIP images.  HEXRD crashes before I can copy the text, but that is the gyst.

**Comment by wehrenberg1** [comment]
oh I forgot, we need to use hdf instead of gel

**Comment by psavery** [comment]
Yes that is right. Maybe gel is a file format we should look into supporting, however? @saransh13

**Comment by saransh13** [comment]
Yeah I think we should add that. `ImageJ` has a plugin for reading that. I wonder if `FabIO` has something similar?

**Comment by bnmajor** [comment]
I'd need to look through old issues and messages with Joel (if they still exist) but I think there was a reason we didn't add support for gel files. I'll update this issue if I figure that out.

**Comment by psavery** [comment]
@saransh13 Can we get an example? If what I was reading was correct, they are basically TIF files that have been modified a little bit (and we will need to perform a conversion to modify them back).

**Comment by psavery** [comment]
From the example I got, it looks like this is correct. I can open them with fabio, or a basic image viewer, but the values are scaled down (I believe square root) so that they fit in a 16-bit integer. They may be multiplied by a constant as well. So we just need to make sure we convert them back after reading it in.

**Comment by psavery** [comment]
I found some source from an ImageJ plugin [here](https://imagej.nih.gov/ij/plugins/download/Linearize_GelData.java)! It looks like it asks the user for a scale factor, and then it squares the data and multiplies by the scale factor.

**Comment by psavery** [comment]
Just to document this here, @saransh13 and I found a way to load them in and convert them.

The basic formula is:
```python
img = raw_img**2 * scale_factor
```

Where, if we use `Pillow` to read in the image, `raw_img = ((2**16 - 1) - pillow_img)`. And if we use fabio to read in the image, it is just `raw_img = fabio_img`.

According to the documentation [here](https://imagej.nih.gov/ij/plugins/download/Linearize_GelData.java), the scale factor is is encoded into a private TIFF tag, but I do not see it. We computed a scale factor, however, of `2.9452155399372724e-07` for a couple of the files we looked at, and we can just go with that. Hopefully that will be the same for all of the files we look at.
